examples/00-mapdl-examples/bracket_static.py

```diff
@@ -0,0 +1,743 @@
+# the fillet line.
+#
+# Select second line for fillet, create python parameter
+line2 = mapdl.lsel("S", "LOC", "X", box2[0])
+l2 = mapdl.get("line2", "LINE", 0, "NUM", "MAX")
+
+###############################################################################
+# Once we have both lines selected we can use the APDL command
+# :meth:`mapdl.lfillt() <ansys.mapdl.core.Mapdl.lfillt>` to generate the fillet
+# between the lines.
+#
+# **Note** that python could return a list if more than one line is selected.
+
+###############################################################################
+# Here we use a mix of the APDL parameter as a string ``line1`` and the ``l2``
+# python parameter to create the fillet line.
+#
+# Create fillet line using selected line (parameter names)
+fillet_radius = 0.4
+mapdl.allsel()
+line3 = mapdl.lfillt("line1", l2, fillet_radius)
+
+mapdl.allsel()
+mapdl.lplot(vtk=True, cpos="xy")
+
+###############################################################################
+# Create fillet area
+# ~~~~~~~~~~~~~~~~~~
+#
+# To create the area delineated by ``line1``, ``line2`` and newly created
+# ``line3``, we use the :meth:`mapdl.al() <ansys.mapdl.core.Mapdl.al>` command.
+# The three lines are the input. If we select them all, we can use the ``'ALL'``
+# argument to create the area.
+#
+# First we have to reselect the newly created lines in the fillet area.
+# To do this we can use the fillet_radius parameter and the
+# :meth:`mapdl.lsel() <ansys.mapdl.core.Mapdl.lsel>` command.
+#
+# For the two newly created straight lines, the length will be the same as the
+# fillet_radius. So we can use the length argument with the
+# :meth:`mapdl.lsel() <ansys.mapdl.core.Mapdl.lsel>` command.
+#
+
+mapdl.allsel()
+
+# Select lines for the area
+mapdl.lsel("S", "LENGTH", "", fillet_radius)
+
+###############################################################################
+# Additionally, we need the fillet line itself (``line3``) but we can use the
+# :meth:`mapdl.lsel() <ansys.mapdl.core.Mapdl.lsel>` command again with either
+# the ``'RADIUS'`` argument if there is only one line with that radius in our model
+# or more directly using the parameter name of the line.
+# Note the ``'A'`` to additionally select items.
+#
+mapdl.lsel("A", "LINE", "", line3)
+
+# plotting ares
+mapdl.lplot(vtk=True, cpos="xy", show_line_numbering=True)
+
+###############################################################################
+# Then we use :meth:`mapdl.al() <ansys.mapdl.core.Mapdl.al>` to create the areas
+# from the lines.
+#
+
+# Create the area
+mapdl.al("ALL")  # Prints the id of the newly created area
+
+
+###############################################################################
+# Add areas together
+# ~~~~~~~~~~~~~~~~~~
+# Append all areas again with the :meth:`mapdl.aadd() <ansys.mapdl.core.Mapdl.aadd>`.
+# Since we have only the 2 areas to combine, use the ``'ALL'`` argument.
```

Comment:
```suggestion
# Append all areas again with the :meth:`mapdl.aadd() <ansys.mapdl.core.Mapdl.aadd> command`.
# Because you have only the two areas to combine, use the ``'ALL'`` argument.
```